### PR TITLE
Rearrange and fix test_assert_email_and_password_length test

### PR DESCRIPTION
**Pull request by bnx05**

### CODE REVIEW

File: test_parameters.py [hunk 1]
```diff
@@ -6,56 +6,63 @@
...
+    browser.find_element_by_name(field_name).click()
+    browser.find_element_by_name(field_name).send_keys(sample_string)
+    assert len(browser.find_element_by_name(field_name).get_attribute("value")) == string_length
+
+
+# this test checks if the login button is enabled after entering different
+# combinations of invalid values in the email and password fields
+@pytest.mark.parametrize("email", email_addresses)
+@pytest.mark.parametrize("password", passwords)
+def test_assert_login_button_enabled(email, password):
```
Comment (marisaq2016) [inline test_parameters.py:48]: this test is no longer asserting the login button

File: test_parameters.py [hunk 2]
```diff
@@ -6,56 +6,63 @@
...
 
 
-@pytest.mark.parametrize("field_name, string_length", [
-    ("login", 12),
-    ("password", 11),
+# this test checks if the values entered into the email field contain '@'
+@pytest.mark.parametrize("email", [
+    "123@abc.org",
+    "info@engagespark.com",
+    pytest.mark.xfail("blah"),
```
Comment (marisaq2016) [inline test_parameters.py]: can we remove ``xfail``?

**Review by marisaq2016 — CHANGES_REQUESTED** [review]

**Comment by bnx05** [comment]
@marisaq2016  updated, please check

**Review by marisaq2016 — APPROVED** [review]
lgtm

**Pull request merged**